### PR TITLE
Feature/dashboard detail view

### PR DESCRIPTION
**Pull request by achen2401**
Create detailed view feature for each account:
https://www.pivotaltracker.com/story/show/172222556

Using this query to get Careplan detail for each user account:
./CarePlan?subject=Patient/[userId]
and then use the id for each Care plan for that subject from that response to query for questionnaire response for the subject: ./QuestionnaireResponse?based-on=[carePlanId]&_include=QuestionnaireResponse:questionnaire
Let me know if that is correct?

### CODE REVIEW

**Review by pbugni — APPROVED** [review]
Nice!  Can't wait to see it :)

File: dashboard/static/src/js/components/Questionnaire.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      this.__mounted = false;
+    }
+    setCurrentState(data) {
+      if (this.__mounted && data) {
+        this.setState(data);
+      }
+    }
+    componentDidMount() {
+        this.__mounted = true;
+        sendRequest(`./QuestionnaireResponse?based-on=${this.props.carePlanId}&_include=QuestionnaireResponse:questionnaire`).then(response => {
```
Comment (ivan-c) [inline dashboard/static/src/js/components/Questionnaire.js]: is there an easy way to store the querystring params as attribute/values in an object? long format strings are harder to read

File: dashboard/static/src/js/components/Table.js [hunk 2]
```diff
@@ -115,6 +128,22 @@ export default class Table extends React.Component {
...
+                            return {
+                              onClick: (e, handleOriginal) => {
+                                //debugging
+                                // console.log("state ", state)
+                                // console.log('A Td Element was clicked!')
+                                // console.log('it produced this event:', e)
+                                // console.log('It was in this column:', column)
+                                // console.log('It was in this row:', rowInfo)
+                                // console.log('row detail: ', rowInfo.row, ' id:', rowInfo.row.id, ' email:', rowInfo.row.email)
+                                // console.log('It was in this table instance:', instance)
```
Comment (ivan-c) [inline dashboard/static/src/js/components/Table.js:141]:
```suggestion
```

File: dashboard/static/src/__tests__/App.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import React from "react";
+import App from '../js/App';
```
Comment (ivan-c) [inline dashboard/static/src/__tests__/App.test.js:3]: Out of scope for this review, but I need your help to refactor the directory structure of the project to make it easier to deploy.

[I created an issue where we can discuss it further](https://github.com/uwcirg/stayhome-dashboard/issues/13)

**Review by ivan-c — APPROVED** [review]
Looks good! Had some nitpicks, but merge when ready.

**Pull request merged**